apps/saleor-app-checkout/config/fields/fields.ts

```diff
@@ -85,6 +87,24 @@ const adyenPaymentProvider: Omit<PaymentProviderSettings<"adyen">, "label">[] =
   },
 ];
 
+const stripePaymentProvider: Omit<PaymentProviderSettings<"stripe">, "label">[] = [
+  {
+    id: "publishableKey",
+    type: "string",
+    encrypt: false,
+  },
+  {
+    id: "secretKey",
+    type: "string",
+    encrypt: true,
+  },
+  {
+    id: "webhookSecret",
```

Comment:
You already have secret and public key. Can't we use them to subscribe to Stripe's events? (Also, user doesn't know which event should subscribe) (The webhook creation would be also the way to validate the provided secret&public key data).  Maybe it is not something for MVP, but definitely, it is a must-have for future releases (configuration PaymentPlugin on Saleor side & then on the payment provider was something that caused a lot of issues).

Reply:
I guess we could create it from the codebase but I'll need to find the best place to do it then :D

Reply:
In Stripe Plugin, we were doing this when the staff user was activating the plugin. I assume that you could do this in a similar way, right?
We store `webhookSecretKey` but we don't show it in plugin configuration form. Instead of this, we show an ID of the webhook instance created by the plugin on Stripe side.
Post MVP sound good, but I would recommend creating at least the task for this

Reply:
https://github.com/saleor/react-storefront/issues/392